components/infobox/commons/infobox_map.lua

```diff
@@ -62,6 +64,8 @@ function Map:_setLpdbData(args)
 		name = self.name,
 		type = 'map',
 		image = args.image,
+		releasedate = args.releasedate,
```

Comment:
doesn't work (line below too)
lpdb datapoint tables do not have those keys
—> move to extradata

Reply:
Moved ``Release Date`` to ``Date``, commented ``game`` out until we have a conclusion to the topic on [#883 discussion](https://github.com/Liquipedia/Lua-Modules/pull/883/files/fd316204f67dd20e7118ed0fd9380ef9eddc21f2#r775282511)